frontend/src/pages/Login.vue

```diff
@@ -101,6 +106,9 @@ export default {
             data: { login },
           } = result
           this.$store.dispatch('login', login)
+          if (this.$route.params.code) {
+            this.redeemLink(this.$route.params.code)
```

Comment:
go to redeem page and ask the user to confirm the link

Reply:
ok